pkg/constants/constants.go

```diff
@@ -72,6 +72,11 @@ const (
 	// auto-label-container-linux=true
 	LabelUpdateAgentEnabled = Prefix + "agent"
 
+	// Label set to "true" on nodes where the update-agent has installed a new
+	// version, and a reboot is needed. Equivalent to the "reboot-needed"
+	// annotation, except that other agents sometimes set the annotation
```

Comment:
huh? "Equivalent except other agents sometimes set the annotation"?

Reply:
I was referencing the behavior of the kvo, but I doubt this is worth mentioning.